### PR TITLE
Use ergoCub face expressions when opening and closing the eyes

### PR DESCRIPTION
**Pull request by S-Dafarra**
This PR allows using https://github.com/icub-tech-iit/ergocub-software/tree/master/src/modules/ergoCubEmotions triggering the animations when the operator closes the eyes

### CODE REVIEW

File: modules/SRanipal_module/src/AdvancedJoypad.cpp [hunk 1]
```diff
@@ -65,9 +66,9 @@ bool AdvancedJoypad::configure(const yarp::os::ResourceFinder &rf, std::shared_p
     return true;
 }
 
-void AdvancedJoypad::setEyeOpenness(double eyeOpenness)
+void AdvancedJoypad::setEyeOpenness(double leftEyeOpennes, double rightEyeOpennes)
 {
-    m_eyeOpennessValue = eyeOpenness;
+    m_eyeOpennessValue = std::min(leftEyeOpennes, rightEyeOpennes);
```
Comment (GiulioRomualdi) [inline modules/SRanipal_module/src/AdvancedJoypad.cpp:71]: Since you get the minimum, is it possibile to control one eye only?

**Review by GiulioRomualdi — APPROVED** [review]

**Comment by S-Dafarra** [comment]
Apparently, in https://github.com/icub-tech-iit/ergocub-software/pull/105 the name of the face expressions have changed

**Comment by GiulioRomualdi** [comment]
Do You want to keep the new neutral when the eyes are closed and allert otherwise?

**Comment by S-Dafarra** [comment]
> Do You want to keep the new neutral when the eyes are closed and allert otherwise?

Yes. I also wanted to make sure that we pushed the modifications to use all the fingers with the gloves

**Comment by S-Dafarra** [comment]
> Do You want to keep the new neutral when the eyes are closed and allert otherwise?

Yes. I also wanted to make sure that we pushed the modifications to use all the fingers with the gloves

**Comment by S-Dafarra** [comment]
> Do You want to keep the new neutral when the eyes are closed and allert otherwise?

Yes. I also wanted to make sure that we pushed the modifications to use all the fingers with the gloves

**Comment by S-Dafarra** [comment]
> Do You want to keep the new neutral when the eyes are closed and allert otherwise?

Done with https://github.com/robotology/walking-teleoperation/pull/127/commits/35b3d780dca8bce724e25fa8b974183160439516



> > Do You want to keep the new neutral when the eyes are closed and allert otherwise?
> 
> Yes. I also wanted to make sure that we pushed the modifications to use all the fingers with the gloves

I checked and everything should be already pushed

@GiulioRomualdi if you agree I would proceed with the merge

**Comment by S-Dafarra** [comment]
> @GiulioRomualdi if you agree I would proceed with the merge

Agreed F2F. Merging

**Pull request merged**